src/platforms/javascript/common/session-replay/troubleshooting.mdx

```diff
@@ -35,6 +35,15 @@ Due to [browser limitations](https://developer.mozilla.org/en-US/docs/Web/SVG/El
 
 </Alert>
 
+## Why can't I see the full HTTP request body or all the headers?
+
+By default, Replay will capture basic information about all outgoing fetch and XHR requests in your application. This includes the URL, request and response body size, method, and status code.
+The intention is to limit the chance of collecting private data. You can <PlatformLink to="/session-replay/configuration/">configure the SDK to capture bodies and additional headers</PlatformLink>.
+
+Another reason you might not see the body is if the format is not supported. We inspect the body and make an educated guess if it is JSON (starts and ends with `[`, `]` or `{`, `}`), otherwise we treat it as a string/array of bytes and take the first 150k bytes. XML is stringified, and will be captured (as well as other strings). `FormData` (and `URLSearchParams`) datatypes are also captured. Bodies for files or images are not captured.
```

Comment:
This is almost correct, _but_ we ignore anything that is not a string. So I think we can omit the `bytes` part here?

Also not sure if we need to explain how we try to identify JSON. IMHO if it _is_ JSON, we will always identify it correctly, and if it isn't, it will never be sent as JSON (as we cannot JSON.parse() it). So maybe:

```suggestion
Another reason you might not see the body is if the format is not supported. We only capture text-based bodies, including JSON, XML, FormData, and similar data types. Any captured body will be truncated to 150k characters. When we identify the body as JSON, we will try to truncate it while keeping the payload valid JSON. Bodies of byte, file, or media types are not captured.
```